### PR TITLE
Move to Azure CI

### PR DESCRIPTION
**Pull request by xhochy**
See https://github.com/conda-forge/conda-forge.github.io/issues/857

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,7 +1,7 @@
 mkdir "%SRC_DIR%"\cpp\build
 pushd "%SRC_DIR%"\cpp\build
 
-cmake -G "%CMAKE_GENERATOR%" ^
+cmake -G "NMake Makefiles" ^
```
Comment (isuruf) [inline recipe/bld.bat]: I suggest using `jom` or `ninja` to make the build parallel.

Comment (xhochy) [reply recipe/bld.bat]: Is `ninja` working nowadays? We had also issues with it in Azure before...

Comment (isuruf) [reply recipe/bld.bat]: Yes. When did you have issues? I fixed some issues with Ninja several months ago.

Comment (xhochy) [reply recipe/bld.bat]: No longer any issues, thanks!

**Comment by kszucs** [comment]
@xhochy nice! We need to back port it to the nightlies.

**Pull request merged**